template/iconComponent.ts

```diff
@@ -0,0 +1,31 @@
+interface templateProps {
+    fileName: string;
+    data: string;
+}
+
+function componentTemplate ({fileName,data}:templateProps) {
+    const render = `
+// GENERATED BY ./scripts/iconScript.ts
+
+import * as React from 'react';
+
+interface iconProps {
+height?: string | number | undefined,
+width?: string | number | undefined,
+color?: string | undefined,
+rotate?: string | undefined
```

Comment:
i think we should only provide number type to `rotate` since we only accept degrees.

Reply:
> i think we should only provide number type to `rotate` since we only accept degrees.

I have pushed a commit that changes the prop type of `rotate` to number